backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepository_deriveSyncToParentStatusUnitTestSuite.java

```diff
@@ -37,7 +38,8 @@ public void branchAndParentPointingSameCommitAndBranchJustCreated_inSync() {
     IGitCoreLocalBranchSnapshot childBranch = createGitCoreLocalBranch(commit);
 
     // when
-    SyncToParentStatus syncToParentStatus = invokeDeriveSyncToParentStatus(childBranch, parentBranch, MISSING_FORK_POINT);
+    SyncToParentStatus syncToParentStatus = invokeDeriveSyncToParentStatus(childBranch, parentBranch,
+        MISSING_FORK_POINT);
```

Comment:
Here and in other places: this lines probably can be joined

Reply:
PR #423 will fix this issue everywhere